app/instance-initializers/validation-i18n.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
```

Comment:
This needs a comment referencing https://github.com/dockyard/ember-validations/issues/224 so we can see where it is coming from and when it can be removed.